packages/components/organisms/f-header/src/components/CountrySelector.vue

```diff
@@ -0,0 +1,287 @@
+<template>
+    <li
+        :class="['c-countrySelector c-nav-list-item has-sublist', {
+            'is-open': countrySelectorIsOpen
+        }]"
+        data-test-id="countrySelector-toggle"
+        v-on="isBelowMid ? null : { mouseover: openCountrySelector, mouseleave: closeCountrySelector }"
+        @keyup.esc="closeCountrySelector">
+        <f-button
+            :tabindex="isBelowMid && !navIsOpen ? -1 : 0"
+            button-type="icon"
+            class="c-nav-list-text c-countrySelector-btn"
+            :aria-expanded="!isBelowMid && countrySelectorIsOpen ? 'true' : 'false'"
+            :aria-haspopup="isBelowMid ? false : true"
+            :aria-label="copy.changeCurrentCountry"
+            @click="onCountrySelectorToggle"
+            v-on="isBelowMid && !countrySelectorIsOpen ? { blur: closeNav, focus: openNav } : null">
```

Comment:
The `isBelowMid && !countrySelectorIsOpen` condition is effectively repeated twice here, as one is the inverse of the other - can you put that into a single `computed` state property?

Reply:
Hmm, @benwhite-deltas I can't see how to put them into a single property...

Reply:
Ah, sorry @dandel10n, I was misremembering my truth tables here. I'd hoped you could do something like:
```
            :aria-expanded="isWideAndExpanded ? 'true' : 'false'"
            :aria-haspopup="isBelowMid ? false : true"
            :aria-label="copy.changeCurrentCountry"
            @click="onCountrySelectorToggle"
            v-on="!isWideAndExpanded ? { blur: closeNav, focus: openNav } : null">
```
But actually `!(isBelowMid && !countrySelectorIsOpen)` == `!isBelowMid || countrySelectorIsOpen` (as per DeMorgan's Theorem). Might still be worth turning these separate values into computed properties to help readers of the code understand the semantics of the conditions, e.g. `isWideViewAndSelectorExpanded` and `isNarrowViewAndSelectorRetracted`, for example.